platform-includes/sourcemaps/overview/javascript.mdx

```diff
@@ -31,6 +31,18 @@ If you used Create React App to set up your React application see our <PlatformL
 - <PlatformLink to="/sourcemaps/uploading/vite/">Vite</PlatformLink>
 - <PlatformLink to="/sourcemaps/uploading/esbuild/">esbuild</PlatformLink>
 
+<Alert>
+  Select the bundler that corresponds to your framework. If you're unsure which bundler your framework uses, check your project's `package.json`, build configuration files, or the framework's documentation.
+</Alert>
+
+<PlatformSection supported={['javascript.cloudflare']}>
+  <Alert>
```

Comment:
Can we avoid double-alerts here? So, can we just put this content in the alert above, when on cloudflare?